### ISSUE TITLE
Cannot compile

### ISSUE THREAD

**Issue by xiafangmo** [mapbox/mapbox-navigation-ios#547]
I'm using Mapbox 3.6.2 and I'm getting Mapbox Navigation from here
```
pod 'MapboxNavigation', :git => 'https://github.com/mapbox/mapbox-navigation-ios.git', :branch => 'master'
```
But I'm getting this error.
<img width="772" alt="screen shot 2017-08-26 at 10 23 29 pm" src="https://user-images.githubusercontent.com/6510543/29746593-62b4389e-8aad-11e7-8495-f6ab7e14ccaf.png">

**Comment by davidseek** [comment]
same. several issues. with the same pod line

**Comment by xiafangmo** [comment]
I restarted the whole thing and got this warning.
- `Mapbox-iOS-SDK (~> 3.6.2)` required by `Podfile`
- `Mapbox-iOS-SDK (= 3.6.1)` required by `MapboxNavigation (0.6.1)`

So now I'm using Mapbox 3.6.1 with Navigation 0.6.1 and now the app compiles fine.

**Comment by bsudekum** [comment]
This should be fixed in the latest release, [v0.7.0](https://github.com/mapbox/mapbox-navigation-ios/releases/tag/v0.7.0).
